packages/engine/src/worker/runner/javascript/error.rs

```diff
@@ -92,6 +92,9 @@ pub enum Error {
 
     #[error("serde: {0:?}")]
     Serde(#[from] serde_json::Error),
+
+    #[error("Couldn't access javascript file {0}: {1}")]
+    MissingJavascriptImport(String, String),
```

Comment:
Maybe use `(PathBuf, String)` instead?

Reply:
I copied `PackageImport` and `FileImport` from the same error.
We only use `String`-like types in the code, we'd have to convert it to a `PathBuf` for the error.

Reply:
Fair enough, let's leave it as `String` then.